CHANGELOG.md

```diff
@@ -0,0 +1,187 @@
+# 0.18.0
+
+Please see https://github.com/rapidsai/dask-cuda/releases/tag/branch-0.18-latest for the latest changes to this development branch.
```

Comment:
Does this mean there will be no summary in `CHANGELOG.md` starting from 0.18?

Reply:
hey @pentschev. That's correct. During development, the changes will be recorded in the GitHub Pre-release that's linked. Then at release time, the body of the pre-release will be copied into this file and committed to the repo.

Reply:
Ah, got it, so it will be done later. Thanks for the info!